manifests/yum.pp

```diff
@@ -43,7 +43,8 @@
 #     ``$servers`` entries appropriately
 #
 class simp::yum (
-  Simplib::Netlist                                     $servers,
+  Integer[1, 1]                                        $api_version         = 1,
```

Comment:
There is already a SemVer data type that supports math operations, though it still needs a SemVer.new() since hiera doesn't automatically use constructors